frontend/src/main/components/Commons/Profits.js

```diff
@@ -24,6 +24,7 @@ const Profits = ({ profits }) => {
                 <Card.Title>
                     You will earn profits from milking your cows everyday at 4am.
                 </Card.Title>
+                <Card.Title>Market Milk Price: ${commons.milkPrice} per healthy cow.</Card.Title>
```

Comment:
This will error and React will remove the UI entirely if the parent component fails to pass `commons` as a prop for any reason, such as http request failing and the parent doesn't conditionally render Profits. You can handle this in several ways:
* (Preferred) just accept milkPrice as the prop. This is more readable and potentially more reusable in the case of parent handling different types of data.
* Conditionally render the milk price card
* Fallback to some text if `commons` is null

Reply:
Or, you could use: 
```
                <Card.Title>Market Milk Price: ${commons?.milkPrice} per healthy cow.</Card.Title>
```

Reply:
This works, but if `commons` is null, the UI will be in a confusing state to users with `Market Milk Price: per healthy cow`. It's better to display a better message like `Milk price failed to load` or remove the row entirely.

Reply:
I opted to use conditional chaining to match the way commons is used in ManageCows.js:
`<Card.Title>Market Cow Price: ${commons?.cowPrice}</Card.Title>`

I think the display message if the milk price fails to load can be a separate PR, as this matches the existing codebase and other components in the commons page can be updated as well.

Reply:
Thanks for fixing, there’s also the argument of not doubling down on UX issues like those for the sake of consistency. End users come first, but it’s a small nitpick.